document-service-api/src/routes/index.js

```diff
@@ -4,8 +4,60 @@
  * This puts all the endpoints into the application.
  */
 
-const { Router } = require('express');
+const {
+    documentStorage: {
+        documentStorage: {
+            directory
+        }
+    }
+} = require('../lib/config');
+const { 
+    DocumentWriter, 
+    DocumentReader 
+} = require('../controllers')
+const { 
+    Router 
+} = require('express');
+const uuid = require('uuid');
+const multer = require('multer');
+const mime = require('mime-types')
 
 const routes = Router();
+const strategy = multer.diskStorage({
```

Comment:
For consistency across packages, I would prefer the `express-file-upload` approach as used in forms-web-app. Also, I don't think that the file upload work should be in the router - I think this should be in the controller and the router merely declares the URL and invokes the controller